site/content/en/docs/tasks/troubleshooting/troubleshooting_provreq.md

```diff
@@ -0,0 +1,224 @@
+```bash
+[...]
+Status:
+  Conditions:
+    Last Transition Time:  2024-05-22T13:04:54Z
+    Message:               Provisioning Request wasn't accepted.
+    Observed Generation:   1
+    Reason:                NotAccepted
+    Status:                False
+    Type:                  Accepted
+    Last Transition Time:  2024-05-22T13:04:54Z
+    Message:               Provisioning Request wasn't provisioned.
+    Observed Generation:   1
+    Reason:                NotProvisioned
+    Status:                False
+    Type:                  Provisioned
+    Last Transition Time:  2024-05-22T13:06:49Z
+    Message:               max cluster limit reached, nodepools out of resources: default-nodepool (cpu, memory)
+    Observed Generation:   1
+    Reason:                OutOfResources
+    Status:                True
+    Type:                  Failed
+```
+
+Note that the `Reason` and `Message` values for `Failed` condition may differ from your output, depending on the
+reason that prevented the provisioning.
+
+The Provisioning Request state is described in the `.conditions[*].status` field.
+An empty field means ProvisinongRequest is still being processed by the ClusterAutoscaler.
+Otherwise, it falls into one of the states listed below:
+- `Accepted` - indicates that the ProvisioningRequest was accepted by ClusterAutoscaler, so ClusterAutoscaler will attempt to provision the nodes for it.
+- `Provisioned` - indicates that all of the requested resources were created and are available in the cluster. ClusterAutoscaler will set this condition when the VM creation finishes successfully.
+- `Failed` - indicates that it is impossible to obtain resources to fulfill this ProvisioningRequest. Condition Reason and Message will contain more details about what failed.
+- `BookingExpired` - indicates that the ProvisioningRequest had Provisioned condition before and capacity reservation time is expired.
+- `CapacityRevoked` - indicates that requested resources are not longer valid.
+
+The states transitions are as follow:
+
+![Provisioning Request's states](/images/prov-req-states.svg)
+
+## Why a Provisioning Request is not created?
+
+If Kueue did not create a Provisioning Request for your job, try checking the following requirements:
+
+### a. Ensure the Kueue's controller manager enables the `ProvisioningACC` feature gate
+
+Run the following command to check whether your Kueue's controller manager has enabled the `ProvisioningACC` feature gate:
+
+```bash
+kubectl describe pod -n kueue-system kueue-controller-manager-
+```
+
+The arguments for Kueue container should be similar to the following:
+
+```bash
+    ...
+    Args:
+      --config=/controller_manager_config.yaml
+      --zap-log-level=2
+      --feature-gates=ProvisioningACC=true
+```
+
+Please note for Kueue `v0.7.0` or newer the feature is enabled by default, so you may see different output.
```

Comment:
Avoid "please" https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language

Same recommendation in other places.